mod/topics/set/type/topic/export.rb

```diff
@@ -7,3 +7,36 @@ def atom
     )
   end
 end
+
+format :jsonld do
+  def molecule
+    {
+      "@context": "#{request.base_url}/context/#{card.type}.jsonld",
+      "@id": path(mark: card.name),
+      "@type": card.type,
+      "name": card.name.parents[1],
+      "description": card.fetch("Overview")&.content,
+      "inScheme": path(mark: card.name.left), 
+      "parent": get_parent,
+      "children": get_children
+    }.compact
+  end
+
+  private
+
+  def get_parent
+    category = card.fetch(:category)
```

Comment:
`category = card.category_card`

(Note, this works because of `card_accessor :category, type: :pointer` in topic.rb)